torch/quantization/quantize.py

```diff
@@ -22,7 +22,8 @@
 
 def is_activation_post_process(module):
     return (isinstance(module, torch.quantization.ObserverBase) or
-            isinstance(module, torch.quantization.FakeQuantizeBase))
+            isinstance(module, torch.quantization.FakeQuantizeBase) or
+            isinstance(module, torch.quantization.Logger))  # type: ignore
```

Comment:
is this the numerical suite logger or something else? what's the context on needing this?

Reply:
this is numeric suite Logger, we need to recognize activation_post_proceess in all_node_args_have_no_tensors after this refactor

Reply:
since we first insert observer and then handle copy node (the handle_copy_nodes function is dealing with a model with observers)

Reply:
btw, sometimes cache doesn't make much sense since the function is called in a local QuantizeHandler

Reply:
makes sense.  What's the context on having to care about NS loggers here, is there a use case?  It would be great to keep quantization code from not knowing anything about numerical suite, if possible.

Reply:
without this it breaks a test in numeric suite

Reply:
which test is it?  I don't see it in CI, can you paste a stack trace?

A `_numerical_suite_fx.Logger` is not an observer, I think we should fix the test instead.  Happy to help with that.  I think it's valuable to keep quantization from knowing anything about Numeric Suite directly, due to separation of concerns.

Reply:
Can we do this in a follow up diff? I have other PRs depending on this one that's up for a while. not sure which test is it, but we can see it if we revert this change

Reply:
it's not about the test I think, it's just that our pass needs to know what is the node that's inserted by prepare so that we can skip them in handle_copy_nodes

Reply:
right, but a Logger is not an observer, so it logically shouldn't be in this function, I don't think we should land this line of code.  If you share the stack trace of the test that's failing, happy to suggest a fix.

Reply:
if there is some test which is running a quantization pass on a model which has Loggers, then that test needs to be fixed.  I think the right behavior for running a quantization pass on a model with Loggers is "fail with an error message".